### PR TITLE
Adds moth seeds to seedvend and various moth food fixes

### PR DESCRIPTION
**Pull request by carshalash**
## About The Pull Request

EoB accidentally forgot to add these to vending machines, this PR throws them in. 

## Why It's Good For The Game

Being able to access berries without calling the admins jannys and begging them for help is nice. 

## Changelog

:cl:
fix: Adds seeds to vending machines as intended. Corrects pathway for lasagne. Fixes issue with Bell peppers and baking cheese.
/:cl:

### CODE REVIEW

**Comment by EOBGames** [comment]
Ah damn, I should have known I forgot something- I had them in the seed vendor on at least one of the three branches I had to work across, so I must have gotten mixed up with that and the final. Thanks for handling that.

The spaghetti in lasagne being cooked was deliberate, though- with dried lasagne sheets you need to boil them first, hence that got translated into the recipe. That said though, if that's too confusing or doesn't convey well, it's probably best with raw pasta.

**Comment by AdipemDragon** [comment]
is.... is the herb bundles handled by this too or.

**Comment by carshalash** [comment]
> is.... is the herb bundles handled by this too or.

Ah shit, I thought those were purposefully made to be ordered at the console. Thanks dragon.

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_moth.dm
Comment (Mothblocks) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_moth.dm:93]: Undocumented change, please update your PR changelog

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Please update your changelog to include what seeds have been added

**Comment by optimumtact** [comment]
> > is.... is the herb bundles handled by this too or.
> 
> Ah shit, I thought those were purposefully made to be ordered at the console. Thanks dragon.

@EOBGames are they?

**Comment by EOBGames** [comment]
> > > is.... is the herb bundles handled by this too or.
> > 
> > 
> > Ah shit, I thought those were purposefully made to be ordered at the console. Thanks dragon.
> 
> @EOBGames are they?

The herbs are intended to be grown at botany, hence why they've got seeds- so adding them to the seed vendor is correct.

**Comment by EOBGames** [comment]
Hey, a bug's come up: would you be able to put this variable:
`burns_in_oven = TRUE`
on the roasted bell pepper, so it doesn't catch fire once it's cooked.
(Yes, I feel stupid for missing that one)

**Comment by AdipemDragon** [comment]
Well if this is just becoming a general "fix the moth food pr" pr, what about the baked cheese wheel? That's another one that's just not workin no matter what.

**Comment by carshalash** [comment]
Pepper done, cheese done, lasagne done, seeds done.

Waiting on eob's approval to touch a recipe ratio then we're good to go.

**Comment by MrMelbert** [comment]
> Waiting on eob's approval to touch a recipe ratio then we're good to go.

Did something come of this, or is this good to go now?

**Pull request merged**
